package.json

```diff
@@ -8,43 +8,40 @@
   },
```

Comment:
changes here are from updated `react-scripts`.
I've also moved some dependencies to `devDependencies`